### DISCUSSION TITLE
[Logging]: Can we have an ability to Log Errors only and not the whole debug?

### DISCUSSION THREAD

**Discussion by Starboy-Xo** [martinrotter/rssguard#645] [Q&A]
According to current logging scenario, the --log feature is able to log every single activity that happens within the application.
Which is space eating and bloating. Although, everything written in it is totally understandable. 
What if, you curate the logging into three categories? Here is the example of them:

1. Log Errors = That logs errors only
2. Log Warning = That logs errors + warnings
3. Log All = That Logs everything like it does now

**If it falls in a feature request category, just tell me, I will convert it to issue myself.**

**Comment by martinrotter** [comment]
You can easily filter the log with `grep` or similar tools on every available OS.

**Reply by martinrotter** [reply]
Added, `--no-debug-output` command switch, which disables JUST debug outputs and keeps warnings and errors ON.

**Reply by martinrotter** [reply]
https://github.com/martinrotter/rssguard/commit/7c96a97503f8f5bf009de89891c5a92c889336f1

**Reply by Starboy-Xo** [reply]
oh thanks a lot!
i'll try it.

**Reply by Starboy-Xo** [reply]
it works and it is LIT!
